apps/fabric-tester/App.tsx

```diff
@@ -75,7 +78,13 @@ export function LinearGradientExample() {
   return (
     <View style={styles.exampleContainer}>
       <View style={styles.gradient}>
-        {mounted && <LinearGradient style={{ flex: 1 }} colors={colors} end={{ x: 0.5, y: 1.0 }} />}
+        {mounted && (
+          <LinearGradient
+            style={{ flex: 1 }}
+            colors={colors as unknown as readonly [string, string, string]}
```

Comment:
just a nit but wouldn't it be easier to type this inside the `useState<>`?

Reply:
added a function that returns correctly typed result which is cleaner, TY :+1: